.circleci/config.yml

```diff
@@ -62,20 +62,6 @@ jobs:
                      --password=$SANDBOX_PASSWORD \
                      --client-id=50 \
                      tests/test_sync_canary.py
-      - run:
-          when: always
-          name: 'Testing Start Date'
```

Comment:
Note this is a duplicate that must have been merged in by 2 people, the test still runs in cirlce